lib/active_model_serializers.rb

```diff
@@ -1,7 +1,18 @@
+module ActiveModelSerializers
+  module_function
+
+  def silence_warnings
+    verbose = $VERBOSE
+    $VERBOSE = nil
+    yield
+  ensure
+    $VERBOSE = verbose
+  end
+
+end
```

Comment:
required because of a Ruby bug or unimportant warnings. such is life

- https://bugs.ruby-lang.org/issues/10967

e.g.

```ruby
  private
 		 
  ActiveModelSerializers.silence_warnings do
    attr_reader :resource, :adapter_opts, :serializer_opts
  end
```